frontend/test/AllQueues.test.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable quotes */
```

Comment:
@hhummel tests look great.  would you be willing to update the front end testing wiki doc that I made with any additional information that you learned thru your process of making these tests?  https://github.com/CodeForPhilly/prevention-point/wiki/Front-End-Automated-Testing

Reply:
Yes